pkg/pg/txdb_test.go

```diff
@@ -0,0 +1,69 @@
+package pg
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/marcboeker/go-duckdb"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"context"
```

Comment:
Does the linter normally fail to flag unsorted std lib? :confused: That is surprising

Reply:
My guess would be that it makes sure they are sorted within the groups you choose, but won't combine different groups if they are separated with a blank line. Lately it seems GoLand keeps inserting imports in weird places, I wonder if there's an adjustment I could make to my settings to get it to choose more wisely.

Reply:
I thought `goimports` would put std lib first, but GoLand certainly has a setting at least